juriscraper/pacer/rss_feeds.py

```diff
@@ -182,18 +184,49 @@ def _get_case_name(self, title_text):
         return case_name
 
 
-if __name__ == "__main__":
-    if len(sys.argv) < 2:
-        print("Usage: python -m juriscraper.pacer.rss_feeds [pacer_court_id] "
-              "[verbose]")
-        print("Please provide a valid PACER court id as your only argument")
-        sys.exit(1)
-    feed = PacerRssFeed(sys.argv[1])
-    print("Querying RSS feed at: %s" % feed.url)
-    feed.query()
-    print("Parsing RSS feed for %s" % feed.court_id)
-    feed.parse()
+def _main():
+    # For help: python -m juriscraper.pacer.rss_feeds -h
+    parser = argparse.ArgumentParser(
+        prog="python -m %s.%s" %
+        (__package__,
+         os.path.splitext(os.path.basename(sys.argv[0]))[0]))
+    parser.add_argument('-b', '--bankruptcy', action='store_true',
+                        help='Use bankruptcy parser variant.')
+    parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument('court_or_file', nargs='?', default='-',
+                        help='''
+A PACER court id or a local filename; defaults to stdin (-).
+Any 3 or 4 character string is presumed to be a court;
+sorry if that was your filename.''')
+
+    args = parser.parse_args()
+
+    arg_len = len(args.court_or_file)
+    if arg_len >= 3 and arg_len <= 4:
+        feed = PacerRssFeed(args.court_or_file)
+        print("Querying RSS feed at: %s" % feed.url)
+        feed.query()
+        print("Parsing RSS feed for %s" % feed.court_id)
+        feed.parse()
+    else:
+        if not args.bankruptcy:
+            feed = PacerRssFeed('fake')
```

Comment:
Since we're giving fake court IDs, we might as well go big, right? How about `fake_court_id` and `fake_court_id_b`?

Reply:
Well, it seems more likely than not that there is or will be code that presumes short court IDs, even if it's only for display purposes. This seems safer.

In the alternative, I guess, since `PacerRssFeed()` is defined in this file, we could just give it a different constructor that doesn't require a feed name and takes a `bankruptcy=True` flag. But I think there are other parts of the juriscraper/CL ecosystem that presume a court ID?

Reply:
I wouldn't want to mess with the constructor just for this. Long court IDs are fine though. Especially in this context.

Reply:
Oh....kay. Well, let's make clear what's district and what's bk then. Done.